### DISCUSSION TITLE
免费节点靠谱吗？访问https网页是否有被节点提供商窃取密码的风险？

### DISCUSSION THREAD

**Discussion by Git-Shaw** [v2fly/v2ray-core#2810] [Q&A]
小白求助，github、youtube有些人提供免费节点，貌似是vless+ws+tls。
有没有风险如访问https网页被节点提供者窃取密码之类的？
![image](https://github.com/v2fly/v2ray-core/assets/61546630/44cce3c6-3e72-4ac4-8ccd-1f7e08663518)

**Comment by tomandjerry27** [comment]
你只要不把别人给的证书加到系统或者浏览器中，你就是安全的。
你用节点的话，节点的提供者只可以知道你在什么时候用什么IP访问了什么网站，再具体的数据他们就无法知道了。所以你用免费节点可能被条子钓鱼执法，但你不会被窃取密码。
